doc/newsfragments/1831_new.driver_log_capture.rst

```diff
@@ -0,0 +1,2 @@
+Added :py:meth:`_record_resource_exception <testplan.common.entity.base.Environment._record_resource_exception>` to fetch error log messages from failed resources during Environment start or stop.
+Added :py:meth:`fetch_error_log <testplan.common.entity.base.Resource.fetch_error_log>` to fetch several lines of text from log files of resources.
```

Comment:
Drivers now can attach some extra logs to the report in case of startup/shutdown failure, with implementing: :py:meth:`fetch_error_log <testplan.common.entity.base.Resource.fetch_error_log>`